src/main/kotlin/no/nb/mlt/wls/product/payloads/SynqProductPayload.kt

```diff
@@ -1,24 +1,86 @@
 package no.nb.mlt.wls.product.payloads
 
+import no.nb.mlt.wls.core.data.Environment
+import no.nb.mlt.wls.core.data.HostName
+import no.nb.mlt.wls.core.data.Owner
+import no.nb.mlt.wls.core.data.Packaging
+import no.nb.mlt.wls.product.model.Product
+import no.nb.mlt.wls.product.payloads.SynqProductPayload.SynqOwner
+import no.nb.mlt.wls.product.payloads.SynqProductPayload.SynqOwner.NB
+import no.nb.mlt.wls.product.payloads.SynqProductPayload.SynqPackaging
+import no.nb.mlt.wls.product.payloads.SynqProductPayload.SynqPackaging.ESK
+import no.nb.mlt.wls.product.payloads.SynqProductPayload.SynqPackaging.OBJ
+
 data class SynqProductPayload(
     val productId: String,
-    val owner: Owner,
+    val owner: SynqOwner,
     val barcode: Barcode,
     val productCategory: String,
     val productUom: ProductUom,
     val confidential: Boolean,
     val hostName: String
 ) {
-    enum class Packaging {
+    enum class SynqPackaging {
         OBJ,
         ESK
     }
 
-    enum class Owner {
+    enum class SynqOwner {
         NB
     }
 
     class Barcode(val barcodeId: String)
 
-    class ProductUom(val uomId: Packaging)
+    class ProductUom(val uomId: SynqPackaging)
+}
+
+// TODO - Will we ever receive this payload?
```

Comment:
Probably no, can either leave it for consistency's sake, or delete it for simplicity's sake